frontend/packages/console-dynamic-plugin-sdk/src/extensions/create-project-modal.ts

```diff
@@ -12,6 +13,8 @@ export type CreateProjectModal = ExtensionDeclaration<
   {
     /** A component to render in place of the create project modal */
     component: CodeRef<ModalComponent<CreateProjectModalProps>>;
+    /** A label to display in the create project dropdown */
+    label: string;
```

Comment:
This label is only used if there is more than one extension, correct? I think we should update the extension point description and this field to explain the behavior more fully.

Reply:
Also worth mentioning that console generates a generic label if none is provided.